text/english/dialog/vccopzam.msg

```diff
@@ -54,29 +54,29 @@
 
 #���� �����
 {160}{}{I'm looking for jay Kukish, a biker. Heard anything about him?}
-{161}{}{Wow - what a coincidence. We are also looking for it! We have orders to detain the offender. If you meet him - remember, he is very dangerous. Do not play around - shoot first, ask questions later.}
+{161}{}{Wow - what a coincidence. We are also looking for him! We have orders to detain the offender. If you meet him - remember, he is very dangerous. Do not play around - shoot first, ask questions later.}
 {162}{}{Uh, thanks for the info.}
 {163}{}{Very well. It was a correct decision.}
 
 #���� �����
-{170}{}{I demand to release Murphy from the isolator.}
-{171}{}{This possibility is allowed in two cases: either you will prove his innocence, or pay a deposit of 500$.}
+{170}{}{I demand that you release Murphy from his cell.}
+{171}{}{You've got two options: either prove his innocence, or pay a deposit of 500$.}
 {172}{}{I will pay. Here's the money.}
 {173}{}{No use talking to you. See you.}
 {174}{}{Prisoner Murphy! You're free to go.}
 {175}{}{Thank you.}
 {176}{}{The money is more important than your health? And if you get infected?}
 {177}{}{Ha-ha-ha, you don't say? This trick is a hundred years old. I've heard everything: "Wasteland disease", "shakes", "Narg syndrome"... What did you come up with this time?}
 {178}{}{You're telling me that, the "Wasteland disease" is not real? I'm a doctor with many years of experience, we have seen the whole villages die from that. Just like in Tonopah or Mesquite. Have you ever been there?}
-{179}{}{Hmm. I heard something about Tonopah. Something about mad people, consuming their own flesh... Or it was really the whole village, which just died out there... But since you're a doctor, I have to believe you. First, write a receipt that you vouch for their diagnosis.}
+{179}{}{Hmm. I heard something about Tonopah. Something about mad people, consuming their own flesh... Or it was really the whole village, which just died out there... But since you're a doctor, I have to believe you. First, write a receipt that you vouch for the diagnosis.}
```

Comment:
"receipt" doesn't make much sense in this context ("affidavit" would be better), but "receipt" is used all throughout Clean City quests and should probably be changes holistically.